setup.py

```diff
@@ -0,0 +1,9 @@
+from setuptools import setup
+
+setup(name="skymapper",
+            version='1.0',
```

Comment:
I love it that you think this is version 1.0; version 0.1 is more appropriate...